File: app/src/main/java/com/example/tsj/service/model/RequestTypeModel.java
package com.example.tsj.service.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RequestTypeModel {
    @SerializedName("RequestTypeId")
    @Expose
    private Integer requestTypeId;
    @SerializedName("RequestTypeName")
    @Expose
    private String requestTypeName;

    public Integer getRequestTypeId() {
        return requestTypeId;
    }

    public void setRequestTypeId(Integer requestTypeId) {
        this.requestTypeId = requestTypeId;
    }

    public String getRequestTypeName() {
        return requestTypeName;
    }

    public void setRequestTypeName(String requestTypeName) {
        this.requestTypeName = requestTypeName;
    }

    @Override
    public String toString() {
        return requestTypeName;
    }
}

File: app/src/main/java/com/timelysoft/tsjdomcom/service/model/RequestTypeModel.kt
package com.timelysoft.tsjdomcom.service.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RequestTypeModel {
    @SerializedName("RequestTypeId")
    @Expose
    var requestTypeId: Int = 0
    @SerializedName("RequestTypeName")
    @Expose
    var requestTypeName: String = ""

    override fun toString(): String {
        return requestTypeName
    }
}